TipCatDotNet.Api/Data/Models/Stripe/StripeAccount.cs

```diff
@@ -1,9 +1,12 @@
+using System;
+
 namespace TipCatDotNet.Api.Data.Models.Stripe;
 
 public class StripeAccount
 {
     public int Id { get; set; }
     public string StripeId { get; set; } = null!;
     public int MemberId { get; set; }
-    // Any account info from stripe
+    public DateTime LastPayout { get; set; }
+    public DateTime LastPayment { get; set; }
```

Comment:
I suggest `Received` or `LastReceived` to avoid usage of 'pay' in both in and out cases